packages/flutter_tools/test/general.shard/resident_web_runner_test.dart

```diff
@@ -79,8 +81,6 @@ const List<VmServiceExpectation> kAttachExpectations = <VmServiceExpectation>[
 ];
 
 void main() {
```

Comment:
I thought it would simplify the debugging if I replaced testbed with testUsingContext ... that was sort of true.